src/messages/message_raw_crypto.rs

```diff
@@ -98,11 +86,19 @@ impl Message {
     /// Decrypts received cypher into instance of `Message`.
     /// Received message should be encrypted with our pub key.
     /// Returns `Ok(Message)` if decryption / deserialization
-    ///     succeeded. `Error` otherwise.
+    /// succeeded. `Error` otherwise.
+    ///
+    /// # Arguments
+    ///
+    /// * `received_message` - received message as byte array
+    ///
+    /// * `decryptor` - decryptor that should be used
```

Comment:
```suggestion
    /// * `decrypter` - decryptor that should be used
```


or rename the parameter?

Reply:
Will rename variables to achieve consistent naming in a manual commit.